src/js/components/InfiniteScroll/__tests__/InfiniteScroll-test.js

```diff
@@ -81,26 +81,59 @@ describe('InfiniteScroll', () => {
   });
 
   test('mixed items', () => {
-    const tempItems = [
-      1,
-      'Hello World',
-      <Image
-        a11yTitle="Gremlin"
-        src="https://v2.grommet.io/img/stak-hurrah.svg"
-      />,
-      99,
-      `Lorem ipsum dolor sit amet, consectetur adipisicing elit,
-      sed do eiusmod temporincididunt ut labore et dolore magna
-      aliqua. Ut enim ad minim veniam, quis nostrud exercitation
-      ullamco laboris nisi ut aliquip ex ea commodo consequat.
-      Duis aute irure dolor in reprehenderit in voluptate velit
-      esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
-      occaecat cupidatat non proident, sunt in culpa qui officia
-      deserunt mollit anim id est laborum.`,
-    ];
+    const lorem = `Lorem ipsum dolor sit amet, consectetur adipisicing elit,
+    sed do eiusmod temporincididunt ut labore et dolore magna
+    aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+    ullamco laboris nisi ut aliquip ex ea commodo consequat.
+    Duis aute irure dolor in reprehenderit in voluptate velit
+    esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+    occaecat cupidatat non proident, sunt in culpa qui officia
+    deserunt mollit anim id est laborum.`;
+    const mixedItems = [];
+    for (let i = 0; i < 1000; i += 1) {
```

Comment:
Probably can nudge this down a bit.
```suggestion
    for (let i = 0; i < 200; i += 1) {
```